prow/plugins/trigger/pull-request.go

```diff
@@ -297,6 +304,13 @@ I understand the commands that are listed [here](https://go.k8s.io/bot-commands?
 	return nil
 }
 
+func draftMsg(ghc githubClient, trigger plugins.Trigger, pr github.PullRequest) error {
+	org, repo, _ := orgRepoAuthor(pr)
+
+	comment := `Skipping CI for Draft PR`
```

Comment:
Not sure if this comment is enough :thinking:

Reply:
Maybe extend with `If you want CI signal for your change, please convert it to an actual PR" (is there a better term for "non-draft-pr" than "actual pr"?)

Reply:
Also tell people that they can manually trigger with `/test all` (that will work, right?)

What is your thinking for using use ``` ` ``` here instead of `"`?

Reply:
I started from the welcomeMsg until i saw that i don't need most of it.
Just forgot to change the `